src/Loaders/LoadingSpinner.tsx

```diff
@@ -73,6 +73,9 @@ export const LoadingSpinner: React.FC<Props> = ({
 
   const pixelSize = SIZE_MAP[size];
 
+  const mountTime = React.useRef(Date.now());
+  const mountDelay = -(mountTime.current % 540);
```

Comment:
We need to disable this for chromatic testing. Fortunately, Chromatic prescribes a way to do this (https://docs.chromaticqa.com/ignoring-elements):

```
import isChromatic from 'storybook-chromatic/isChromatic';
```

You can check the source and see that it checks the user agent, which is a trivial amount of code to add so I'm ok with it.

```js
const mountDelay = isChromatic() ? 0 : -(mountTime.current % 540);
```